src/components/PopupWithForm.js

```diff
@@ -5,12 +5,6 @@ export default class PopupWithForm extends Popup {
     super(popupSelector);
     this._popupForm = this._popupElement.querySelector(".modal__form");
     this._handleFormSubmit = handleFormSubmit;
-    this._profileDescription = document.querySelector(".profile__subheading");
-    this._profileName = document.querySelector(".profile__title");
-    this._modalInputName = document.querySelector("#profileModal__input-name");
-    this._modalInputDescription = document.querySelector(
-      "#profileModal__input-description"
-    );
     this._inputList = [...this._popupForm.querySelectorAll(".modal__input")];
     this._closeButtonPopup = this._popupElement.querySelector(".modal__close");
```

Comment:
`    this._closeButtonPopup = this._popupElement.querySelector(".modal__close");
`

Please, delete `this._closeButtonPopup` because you don't use it